packages/docs/.vuepress/components/ods-icon.vue

```diff
@@ -0,0 +1,29 @@
+<!-- Usage: <ods-icon icon="complete" /> -->
+
+<template>
+  <div class="ods-icon" v-html="require(`!html-loader!@okta/odyssey-icons/src/${icon}.svg`)" />
+</template>
+
+<style lang="scss">
+.ods-icon {
+  display: inline-block;
+
+  svg {
+    width: 24px;
+    height: 24px;
```

Comment:
In general, we should try to specify anything that could/may be customized as props to avoid hardcoding in assumptions. Size is an excellent example where we're likely going to need variability.

Reply:
This is something @edburyenegren-okta will address in a follow up PR since he is closer to the Icon work than I am. I only created this icon because I had a need in a different working branch. He will ensure that the props/styles are reflected based on the design guidelines